docker-vms/jssandbox/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:2.7.9
```

Comment:
We should probably update to python 2.7.10 now, I guess (not in this PR probably).